File: 이건희/SWexpertacademy/D3/4406.py
T = int(input())
for test_case in range(1, T + 1):
    word = str(input())
     
    word = word.replace('a', '').replace('e', '').replace('i', '').replace('o', '').replace('u', '')
             
    print(f'#{test_case} {word}')

File: 이훈일/삼성 SW 역량 테스트 기출 문제/4406.java
import java.util.Scanner;
import java.io.FileInputStream;

class Solution {
	public static void main(String args[]) throws Exception {
	
		Scanner sc = new Scanner(System.in);
		int T;
		T=sc.nextInt();
		sc.nextLine();
        
		for(int test_case = 1; test_case <= T; test_case++) {
            String line = sc.nextLine();
            String replace = line.replaceAll("[aeiou]","");
            System.out.println("#" + test_case + " " + replace);
		}
	}
}
